build.sbt

```diff
@@ -1,9 +1,15 @@
 name := "circe-yaml"
-version in ThisBuild := "0.3.1"
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
This is possibly off-topic, but what do you think about following the sbt-release convention of putting this in a separate `version.sbt`? I think it makes sense and is convenient even for projects that aren't using sbt-release.